components/icons/EmailIcon.tsx

```diff
@@ -0,0 +1,7 @@
+import Ionicons from '@expo/vector-icons/Ionicons'
+
+function EmailIcon() {
+  return <Ionicons name='md-mail' size={20} color='white' />
```

Comment:
evitar usar colores predeterminados, en su lugar usar hexadecimales : 
```suggestion
  return <Ionicons name='md-mail' size={20} color='#ffffff' />
```

Reply:
realizar esto en todos los iconos agregados